app/kumactl/pkg/entities/entities.go

```diff
@@ -0,0 +1,37 @@
+package entities
+
+import (
+	"io"
+	"time"
+
+	core_system "github.com/kumahq/kuma/pkg/core/resources/apis/system"
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+)
+
+type resourceFormatter func(rootTime time.Time, resources model.ResourceList, out io.Writer) error
+type Definition struct {
+	Singular     string
+	Plural       string
+	ResourceType model.ResourceType
+	Formatter    resourceFormatter
+	ReadOnly     bool
+}
+
+var All []Definition
+var Names []string
+var ByName = map[string]Definition{}
+
+func init() {
+	All = append(All, meshEntities...)
+	All = append(All, systemEntities...)
+	All = append(All, Definition{Singular: "global-secret", Plural: "global-secrets", ResourceType: core_system.GlobalSecretType, Formatter: BasicGlobalResourceTablePrinter})
+	for i := range All {
+		// Preserve backward compatibility for this typo before this was auto generated
+		if All[i].Singular == "health-check" {
+			All[i].Plural = "healthchecks"
+			All[i].Singular = "healthcheck"
```

Comment:
`proxytemplate` is wrong in the same way?

Reply:
Proxy template is wrong is a different way:
- Healthcheck has a rest endpoint: `health-check` and a kumactl `healthcheck`
- Proxytemplate: has both rest and kumactl to `proxytemplate`

So for proxytemplate while it's incoherent with how we right things it's coherent between rest and kumactl so no need to do an exception here.